# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/fenced-frame/csp-blocked.https.html.ini
# Commit: e9ee515307b2
# Full Hash: e9ee515307b2acef4ce4ad88ca2e9d8a9aa781a8
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2023-04-23 09:12:41
# Regressor Bug: 1829397
# File Overlap Count: 3
# Description:
#   Bug 1829397 [wpt PR 39645] - Update wpt metadata, a=testonly
#   
#   wpt-pr: 39645
#   wpt-type: metadata
# ==============================================================================

diff -r 73832b4dfc98 -r e9ee515307b2 testing/web-platform/meta/fenced-frame/csp-blocked.https.html.ini
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/meta/fenced-frame/csp-blocked.https.html.ini	Fri Apr 21 19:57:56 2023 +0000
@@ -0,0 +1,78 @@
+[csp-blocked.https.html]
+  [Fenced frame blocked for CSP fenced-frame-src 'none' using config]
+    expected: FAIL
+
+  [Opaque-ads can load API returns false for 'none' using config]
+    expected: FAIL
+
+  [Fenced frame blocked for CSP fenced-frame-src 'self' using config]
+    expected: FAIL
+
+  [Opaque-ads can load API returns false for 'self' using config]
+    expected: FAIL
+
+  [Fenced frame blocked for CSP fenced-frame-src data: using config]
+    expected: FAIL
+
+  [Opaque-ads can load API returns false for data: using config]
+    expected: FAIL
+
+  [Fenced frame blocked for CSP fenced-frame-src https://* using config]
+    expected: FAIL
+
+  [Opaque-ads can load API returns false for https://* using config]
+    expected: FAIL
+
+  [Fenced frame blocked for CSP fenced-frame-src https://*:80 using config]
+    expected: FAIL
+
+  [Opaque-ads can load API returns false for https://*:80 using config]
+    expected: FAIL
+
+  [Fenced frame blocked for CSP fenced-frame-src https://b.test:* using config]
+    expected: FAIL
+
+  [Opaque-ads can load API returns false for https://b.test:* using config]
+    expected: FAIL
+
+  [Fenced frame not loaded using config if any of CSPs in place disallow loading]
+    expected: FAIL
+
+  [Fenced frame blocked for CSP fenced-frame-src 'none' using urn:uuid]
+    expected: FAIL
+
+  [Opaque-ads can load API returns false for 'none' using urn:uuid]
+    expected: FAIL
+
+  [Fenced frame blocked for CSP fenced-frame-src 'self' using urn:uuid]
+    expected: FAIL
+
+  [Opaque-ads can load API returns false for 'self' using urn:uuid]
+    expected: FAIL
+
+  [Fenced frame blocked for CSP fenced-frame-src data: using urn:uuid]
+    expected: FAIL
+
+  [Opaque-ads can load API returns false for data: using urn:uuid]
+    expected: FAIL
+
+  [Fenced frame blocked for CSP fenced-frame-src https://* using urn:uuid]
+    expected: FAIL
+
+  [Opaque-ads can load API returns false for https://* using urn:uuid]
+    expected: FAIL
+
+  [Fenced frame blocked for CSP fenced-frame-src https://*:80 using urn:uuid]
+    expected: FAIL
+
+  [Opaque-ads can load API returns false for https://*:80 using urn:uuid]
+    expected: FAIL
+
+  [Fenced frame blocked for CSP fenced-frame-src https://b.test:* using urn:uuid]
+    expected: FAIL
+
+  [Opaque-ads can load API returns false for https://b.test:* using urn:uuid]
+    expected: FAIL
+
+  [Fenced frame not loaded using urn:uuid if any of CSPs in place disallow loading]
+    expected: FAIL